### PR TITLE
Add build instructions to README

### PR DESCRIPTION
**Pull request by data-henrik**
addresses #5

### CODE REVIEW

File: secure-import/README.md [hunk 1]
```diff
@@ -27,4 +27,7 @@ A lightweight tool to encrypt nonce with AES-GCM or AES-CBC encryption on a nonc
     {
         "encryptedNonce": "DVy/Dbk37X8gSVwRA5U6vrHdWQy8T2ej+riIVw==",
         "iv": "puQrzDX7gU1TcTTx"
-    }
\ No newline at end of file
+    }
+    
+    ## Build / run from source
+    You need a [Go](https://golang.org/) build environment and the source file [encrypt.go](encrypt.go) from this directory. Either directly execute `go run encrypt.go` to run the code or first build (`go build -o kms-encrypt-nonce -i encrypt.go) the executable, then run it with parameters as shown above.
```
Comment (ccbarragan) [inline secure-import/README.md]:
```suggestion
## Build / run from source
You need a [Go](https://golang.org/) build environment and the source file [encrypt.go](encrypt.go) from this directory. Either directly execute `go run encrypt.go` to run the code or first build (`go build -o kms-encrypt-nonce -i encrypt.go) the executable, then run it with parameters as shown above.
```

**Review by ccbarragan — CHANGES_REQUESTED** [review]
Thanks! Small formatting fix, otherwise looks good to me. 

I am no longer a maintainer for this service, but sending this over to Key Protect folks to review.

File: secure-import/README.md [hunk 2]
```diff
@@ -27,4 +27,7 @@ A lightweight tool to encrypt nonce with AES-GCM or AES-CBC encryption on a nonc
     {
         "encryptedNonce": "DVy/Dbk37X8gSVwRA5U6vrHdWQy8T2ej+riIVw==",
         "iv": "puQrzDX7gU1TcTTx"
-    }
\ No newline at end of file
+    }
+
```
Comment (cloudpitchford) [inline secure-import/README.md]: On line 31, does there need to be a closing "fence" (\`\`\`) for the code block after the example?

**Review by cloudpitchford — CHANGES_REQUESTED** [review]
I think I see only one formatting change that could affect display.

**Review by cloudpitchford — APPROVED** [review]

**Pull request merged**